src/rustllvm/llvm-auto-clean-trigger

```diff
@@ -1,4 +1,4 @@
 # If this file is modified, then llvm will be forcibly cleaned and then rebuilt.
 # The actual contents of this file do not matter, but to trigger a change on the
 # build bots then the contents should be changed so git updates the mtime.
-2016-07-25b
+2016-07-25c
```

Comment:
Oh you can actually just change this to today's date, the `b` was just because it was modified twice in the same day